modules/ossm-mixer-policy.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-mixer-policy_{context}"]
+
+= Updating Mixer policy enforcement
+In previous versions of {ProductName}, Mixer’s policy enforcement was enabled by default. Mixer policy enforcement is now disabled by default. You must enable it before running policy tasks.
+
+
+.Procedure
+
+. Run this command to check the current Mixer policy enforcement status:
++
+----
+$ oc get cm -n istio-system istio -o jsonpath='{.data.mesh}' | grep disablePolicyChecks
+----
+
+. If `disablePolicyChecks: true`, edit the {ProductShortName} ConfigMap:
++
+
```

Comment:
Is this section rendering correctly?

Reply:
If it didn't before it does now and we verified it together. :)